core/templates/dev/head/pages/email_dashboard/EmailDashboard.js

```diff
@@ -17,8 +17,10 @@
  */
 
 oppia.controller('EmailDashboard', [
-  '$rootDirective', '$scope', 'EmailDashboardDataService', 'UserService',
-  function($rootDirective, $scope, EmailDashboardDataService, UserService) {
+  '$rootDirective', '$rootScope', '$scope', 'EmailDashboardDataService',
```

Comment:
Actually I don't think $rootDirective is used anywhere.

/cc @oppia/dev-workflow-team FYI

Reply:
Yes, it is not used. Shall I remove it then?

Reply:
Yes, please.

/cc @oppia/dev-workflow-team This should be a lint check.

Reply:
Done.